invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -73,25 +72,36 @@ class CreatibutorsFieldForm extends Component {
     const creatibutorsError =
       error || (creatibutorsList === formikInitialValues && initialError);
 
+    let className = "";
+    if (creatibutorsError) {
+      className =
+        typeof creatibutorsError !== "string" ? creatibutorsError.severity : "error";
+    }
+
+    let generalCreatibutorsErrorMessage;
+    if (typeof creatibutorsError === "string") {
+      generalCreatibutorsErrorMessage = creatibutorsError;
+    } else if (typeof creatibutorsError === "object") {
+      generalCreatibutorsErrorMessage = creatibutorsError?.message;
+    }
```

Comment:
I think this one needs a comment on why we do this and what is "general" error
also to improve readability:
```suggestion
    let generalCreatibutorsErrorMessage = contributorsError;
    if (typeof creatibutorsError === "object") {
      generalCreatibutorsErrorMessage = creatibutorsError?.message;
    }
```